pkg/kwokctl/runtime/kind/kind.yaml.tpl

```diff
@@ -104,7 +104,11 @@ nodes:
   - hostPath: {{ .ConfigPath }}
     containerPath: /etc/kwok/kwok.yaml
     readOnly: true
-
+{{ range .ExtraLogMounts }}
```

Comment:
#439 I will add `ExtraVolumes` for additional mount directories. Please review it if you like, it should work here.

Reply:
Oh, of course! I'll this issue on my PR, but having your PR also helps a log :smile: